src/SourceLocator/Ast/Locator.php

```diff
@@ -59,15 +60,20 @@ public function findReflection(Reflector $reflector, LocatedSource $locatedSourc
      * @param LocatedSource $locatedSource
      * @param IdentifierType $identifierType
      * @return \BetterReflection\Reflection\Reflection[]
+     * @throws Exception\AstParse
      */
     public function findReflectionsOfType(Reflector $reflector, LocatedSource $locatedSource, IdentifierType $identifierType)
     {
-        return $this->findReflectionsInTree->__invoke(
-            $reflector,
-            $this->parser->parse($locatedSource->getSource()),
-            $identifierType,
-            $locatedSource
-        );
+        try {
+            return $this->findReflectionsInTree->__invoke(
+                $reflector,
+                $this->parser->parse($locatedSource->getSource()),
+                $identifierType,
+                $locatedSource
+            );
+        } catch (\Exception $exception) {
```

Comment:
is `\Exception` intended here? Are there no more specific exceptions? What about `\Throwable`, if your intent is to catch everything?